dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityPreviewMode.java

```diff
@@ -0,0 +1,89 @@
+package com.dotcms.rendering.velocity.servlet;
+
+import com.dotcms.rendering.velocity.services.PageContextBuilder;
+import com.dotcms.rendering.velocity.services.VelocityType;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.rendering.velocity.services.VelocityType'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)